packages/server/src/api/controllers/row/index.ts

```diff
@@ -100,49 +107,83 @@ export async function find(ctx: any) {
   })
 }
 
-export async function destroy(ctx: UserCtx<DeleteRowRequest>) {
-  const appId = ctx.appId
-  const inputs = ctx.request.body
+function isDeleteRows(input: any): input is DeleteRows {
+  return input.rows !== undefined && Array.isArray(input.rows)
+}
 
+function isDeleteRow(input: any): input is DeleteRow {
+  return input._id !== undefined
+}
+
+async function processDeleteRowRequest(ctx: UserCtx<DeleteRowRequest>) {
```

Comment:
Might be clearer as `processDeleteRowsRequest` as this is always going to be the plural form